optimize/what-is-optimize.md

```diff
@@ -0,0 +1,33 @@
+---
+id: what-is-optimize
+title: What is Optimize?
+description: "Leverage process data and analyze areas for improvement."
+---
+
+:::note
+New to Optimize? Visit our introductory guide to [Optimize]($docs$/guides/improve-processes-with-optimize/) to get started.
```

Comment:
🔗 This uses the MDX plugin I wrote in (TODO: PR number) to have the site interpret which version of the main docs to send the user to, based on the location of the source file.